libPatches.py

```diff
@@ -290,5 +290,296 @@ def get_lib_patches():
+                    }
+                },
+                "name": "net.java.jinput:jinput:2.0.5",
+            },
+            {
+                "downloads": {
+                    "artifact": {
+                        "sha1": "e12fe1fda814bd348c1579329c86943d2cd3c6a6",
+                        "size": 7508,
+                        "url": "https://libraries.minecraft.net/net/java/jutils/jutils/1.0.0/jutils-1.0.0.jar",
+                    }
+                },
+                "name": "net.java.jutils:jutils:1.0.0",
+            },
+            {
+                "downloads": {
+                    "artifact": {
+                        "sha1": "b04f3ee8f5e43fa3b162981b50bb72fe1acabb33",
+                        "size": 22,
+                        "url": "https://libraries.minecraft.net/org/lwjgl/lwjgl/lwjgl-platform/2.9.4-nightly-20150209/lwjgl-platform-2.9.4-nightly-20150209.jar",
+                    },
+                    "classifiers": {
+                        "natives-linux": {
+                            "sha1": "ba120169db0180f4188ddf4b240668a41f11d35f",
+                            "size": 480907,
+                            "url": "https://github.com/r58Playz/lwjgl2-m1/raw/linux-aarch64-built/lwjgl-platform-2.9.4-nightly-20150209-natives-linux.jar",
+                        }
+                    },
+                },
+                "extract": {"exclude": ["META-INF/"]},
+                "name": "org.lwjgl.lwjgl:lwjgl-platform:2.9.4-nightly-20150209-mmachina.2",
+                "natives": {"linux": "natives-linux"},
+            },
+            {
+                "downloads": {
+                    "artifact": {
+                        "sha1": "697517568c68e78ae0b4544145af031c81082dfe",
+                        "size": 1047168,
+                        "url": "https://libraries.minecraft.net/org/lwjgl/lwjgl/lwjgl/2.9.4-nightly-20150209/lwjgl-2.9.4-nightly-20150209.jar",
+                    }
+                },
+                "name": "org.lwjgl.lwjgl:lwjgl:2.9.4-nightly-20150209",
+            },
+            {
+                "downloads": {
+                    "artifact": {
+                        "sha1": "d51a7c040a721d13efdfbd34f8b257b2df882ad0",
+                        "size": 173887,
+                        "url": "https://libraries.minecraft.net/org/lwjgl/lwjgl/lwjgl_util/2.9.4-nightly-20150209/lwjgl_util-2.9.4-nightly-20150209.jar",
+                    }
+                },
+                "name": "org.lwjgl.lwjgl:lwjgl_util:2.9.4-nightly-20150209",
+            },
+            {
+                "downloads": {
+                    "artifact": {
+                        "sha1": "cbac1b8d30cb4795149c1ef540f912671a8616d0",
+                        "size": 128801,
+                        "url": "https://build.lwjgl.org/release/3.3.1/bin/lwjgl-glfw/lwjgl-glfw.jar",
+                    }
+                },
+                "name": "org.lwjgl:lwjgl-glfw:3.3.1",
```

Comment:
Using lwjgl 3.3.1 will probably break some mods + idk if it'll work on 1.16.5-.

Reply:
Yes, we probably need patched LWJGL.

ManyMC uses [this fork](https://github.com/MinecraftMachina/lwjgl3)

Reply:
Why can't we use the fork for Linux too

Reply:
also idk it would be less tested, at the end this lwjgl is what people with raspberry pi were using to this day

Reply:
Added LWJGL 3.2.2 from gman fork as well